### PR TITLE
Fixing the build with VisRec API fixes

### LINKED ISSUE #36: Build failing with errors
Build is failing and hitting with below attached error. Please help in resolving the same!
screen shot 2018-12-11 at 9 37 37 pm

### PR DESCRIPTION
**Pull request by markstur**
Fixes #36

### CODE REVIEW

**Comment by markstur** [comment]
@RiyaMRoy04  Can you test this?  I can build but have issues testing it.

**Comment by christiancompton** [comment]
I have modified the PR to reflect conventional [swift sytle guidelines](https://github.com/realm/SwiftLint/issues/1518) using a `.swiftlint`.  I have also added an optimistic operator to the Cartfile to prevent future major releases from breaking the [Apple Console Starter Kit](https://console.bluemix.net/developer/appledevelopment/starter-kits/visual-recognition-with-core-ml-and-arkit-for-ios-with-watson).

**Review by markstur — COMMENTED** [review]
LGTM, thanks!

File: ios_swift/ResumeARStarter/ViewController.swift [hunk 1]
```diff
@@ -112,23 +111,32 @@ class ViewController: UIViewController, ARSCNViewDelegate {
...
                 }
-                self.isTraining = false;
-            }else{
-
-                self.visualRecognition?.listClassifiers(){
-                classifiers in
+                self.isTraining = false
+            } else {
+                self.visualRecognition?.listClassifiers { response, error in
+                    if let error = error {
```
Comment (stevemar) [inline ios_swift/ResumeARStarter/ViewController.swift:121]: thanks for updating this!

File: ios_swift/ResumeARStarter/ViewController.swift [hunk 2]
```diff
@@ -307,22 +310,31 @@ class ViewController: UIViewController, ARSCNViewDelegate {
...
-            let failure = { (error: Error) in print(error) }
             guard let visualRecognition = self.visualRecognition else {
                 print("No handle on visual recognition service")
                 observer.onCompleted()
                 return Disposables.create()
             }
-            visualRecognition.classifyWithLocalModel(image: uiImage, classifierIDs: self.classifierIds, threshold: 0, failure: failure) { classifiedImages in
-                  observer.onNext((classes: classifiedImages.images, position: worldCoord, frame: frame))
+
+            visualRecognition.classifyWithLocalModel(image: uiImage, classifierIDs: self.classifierIds, threshold: 0) { classifiedImages, error in
```
Comment (stevemar) [inline ios_swift/ResumeARStarter/ViewController.swift:319]: again, thanks for this.

**Review by stevemar — APPROVED** [review]

**Pull request merged**